test/physics/base/Particle.test.cc

```diff
@@ -80,6 +85,59 @@ TEST_F(ParticleTest, params_accessors)
     EXPECT_EQ(PDGNumber(11), defs.id_to_pdg(ParticleId(0)));
 }
 
+//---------------------------------------------------------------------------//
+// IMPORT PARTICLE DATA TEST
+//---------------------------------------------------------------------------//
+
+class ParticleImportTest : public celeritas::Test
+{
+  protected:
+    void SetUp() override
+    {
+        root_filename_ = this->test_data_path("io", "geant-exporter-data.root");
+        RootImporter import_from_root(root_filename_.c_str());
+        data_ = import_from_root("geant4_data", "ImportData");
+        ;
+    }
+    std::string root_filename_;
+    ImportData  data_;
+};
+
+TEST_F(ParticleImportTest, TEST_IF_CELERITAS_USE_ROOT(import_particle))
+{
+    const auto particles = ParticleParams::from_import(data_);
+
+    // Check electron data
+    ParticleId electron_id = particles->find(PDGNumber(11));
+    ASSERT_GE(electron_id.get(), 0);
+    const auto& electron = particles->get(electron_id);
+    EXPECT_SOFT_EQ(0.510998910, electron.mass().value());
+    EXPECT_EQ(-1, electron.charge().value());
+    EXPECT_EQ(0, electron.decay_constant());
+    // Check all names/PDG codes
+    std::vector<std::string> loaded_names;
+    std::vector<int>         loaded_pdgs;
+    for (auto particle_id : range(ParticleId{particles->size()}))
+    {
+        loaded_names.push_back(particles->id_to_label(particle_id));
+        loaded_pdgs.push_back(particles->id_to_pdg(particle_id).get());
+    }
+
+    // Particle ordering is the same as in the ROOT file
```

Comment:
This comment is no longer accurate, right? The ROOT file now stores them in Geant4's internal ordering, and the `ParticleParams::from_import` reorders them.